tests/unit/hive_metastore/test_table_migrate.py

```diff
@@ -604,8 +657,8 @@ def test_revert_report(ws, capsys):
     table_migrate = get_table_migrator(backend)
     table_migrate.print_revert_report(delete_managed=True)
     captured = capsys.readouterr()
-    assert "test_schema1|1|0|1|0|1|0|0|" in captured.out.replace(" ", "")
-    assert "test_schema2|1|0|0|0|0|0|0|" in captured.out.replace(" ", "")
+    assert "test_schema1|1|0|0|1|0|1|0|0|" in captured.out.replace(" ", "")
+    assert "test_schema2|1|0|0|0|0|0|0|0|" in captured.out.replace(" ", "")
```

Comment:
I know this code exists before, but it is impossible to understand what we are asserting without a comment

Reply:
To be honest, I don't fully understand this test code. But my understanding is it iterates all `What` Enum and print the count of migrated table in each `What`. My PR added a new `What.EXTERNAL_HIVESERDE` I need to modify this test to add one more "|0|" to indicate this test migrated 0 `What.EXTERNAL_HIVESERDE`. 
We will need @FastLee to add comment for this test.